libraries/AP_HAL_ChibiOS/hwdef/SVehicle-E2/hwdef.dat

```diff
@@ -0,0 +1,343 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the S-Vehicle E2-plus AutoPilot hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_SVehicle-E2
+
+FLASH_RESERVE_START_KB 128
+
+# to be compatible with the px4 bootloader we need
+# to use a different RAM_MAP
+env USE_ALT_RAM_MAP 1
+
+# flash size
+FLASH_SIZE_KB 2048
+
+env OPTIMIZE -O2
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART3 OTG2
+
+# default to all pins low to avoid ESD issues
+DEFAULTGPIO OUTPUT LOW PULLDOWN
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+PA9 VBUS INPUT OPENDRAIN
+
+# pins for SWD debugging
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# telem1
+PE8 UART7_TX UART7
+PF6 UART7_RX UART7
+PF8 UART7_RTS UART7
+PE10 UART7_CTS UART7
+
+# telem2
+PC8 UART5_RTS UART5
+PC9 UART5_CTS UART5
+PC12 UART5_TX UART5
+PD2 UART5_RX UART5
+
+# telem3
```

Comment:
need to add defualt ptotocol define to MAVLink2 since its supposed to be a telem and had RS422 driver I/O instead of TTL/CMOS and its SERIAL5 which is normally NONE